index.js

```diff
@@ -99,6 +110,14 @@ class Ky {
 			...otherOptions
 		};
 
+		if (supportsAbortController) {
+			if (this._options.signal) {
+				this._options.signal.onabort = () => isAborted && abortController.abort();
```

Comment:
✖  115:5   Prefer addEventListener over onabort  unicorn/prefer-add-event-listener

Reply:
The AbortSignal interface  inherits properties from  EventTarget.
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget
`this._options.signal.addEventListener('abort', () => ... )`

Reply:
Oh didn't know. Pushing a fix now